CMakeLists.txt

```diff
@@ -282,143 +307,152 @@ set(ARCHIVE_OUTPUT_DIRECTORY ./bin)
 set(CMAKE_CXX_FLAGS "-fPIC -Wall -Werror -Wuninitialized -Wvla")
 
 if(USE_CCACHE)
-  # Configure CCache if available
-  find_program(CCACHE_FOUND ccache)
-  if(CCACHE_FOUND)
-    get_property(RULE_LAUNCH_COMPILE GLOBAL PROPERTY RULE_LAUNCH_COMPILE)
-    set(RULE_LAUNCH_COMPILE "${RULE_LAUNCH_COMPILE} ccache")
-    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ${RULE_LAUNCH_COMPILE})
-
-    get_property(RULE_LAUNCH_LINK GLOBAL PROPERTY RULE_LAUNCH_LINK)
-    set(RULE_LAUNCH_LINK "${RULE_LAUNCH_LINK} ccache")
-    set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK ${RULE_LAUNCH_LINK})
-
-    message(INFO " Used CCache for compilation.")
-  else(CCACHE_FOUND)
-    message(WARNING " CCache was not found.")
-  endif(CCACHE_FOUND)
+    # Configure CCache if available
+    find_program(CCACHE_FOUND ccache)
+    if(CCACHE_FOUND)
+        get_property(RULE_LAUNCH_COMPILE GLOBAL PROPERTY RULE_LAUNCH_COMPILE)
+        set(RULE_LAUNCH_COMPILE "${RULE_LAUNCH_COMPILE} ccache")
+        set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ${RULE_LAUNCH_COMPILE})
+
+        get_property(RULE_LAUNCH_LINK GLOBAL PROPERTY RULE_LAUNCH_LINK)
+        set(RULE_LAUNCH_LINK "${RULE_LAUNCH_LINK} ccache")
+        set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK ${RULE_LAUNCH_LINK})
+
+        message(INFO " Used CCache for compilation.")
+    else(CCACHE_FOUND)
+        message(WARNING " CCache was not found.")
+    endif(CCACHE_FOUND)
 endif(USE_CCACHE)
 
 if(USE_DISTCC)
-  # Configure distributed compilation if available
-  find_program(DISTCC_FOUND distcc)
-  if(DISTCC_FOUND)
-    get_property(RULE_LAUNCH_COMPILE GLOBAL PROPERTY RULE_LAUNCH_COMPILE)
-    set(RULE_LAUNCH_COMPILE "${RULE_LAUNCH_COMPILE} distcc")
-    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ${RULE_LAUNCH_COMPILE})
-    message(INFO " Used distributed build.")
-  else(CCACHE_FOUND)
-    message(
-      WARNING " distcc was not found. Distributed compilation is impossible.")
-  endif(DISTCC_FOUND)
+    # Configure distributed compilation if available
+    find_program(DISTCC_FOUND distcc)
+    if(DISTCC_FOUND)
+        get_property(RULE_LAUNCH_COMPILE GLOBAL PROPERTY RULE_LAUNCH_COMPILE)
+        set(RULE_LAUNCH_COMPILE "${RULE_LAUNCH_COMPILE} distcc")
+        set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ${RULE_LAUNCH_COMPILE})
+        message(INFO " Used distributed build.")
+    else(CCACHE_FOUND)
+        message(
```

Comment:
@vilyukh bring to the form as
```
message(WARNING
                "distcc was not found. Distributed compilation is impossible." 
)
```

Reply:
@BSolonenko You can correct it, should run cmake-format with follow argument: --dangle-parens 1 or add in config-file as dangle_parens: True

Reply:
@MaximRybak please make this default